--- a/docs/conf.py	2023-03-23 09:30:01.556715659 +0200
+++ b/docs/conf.py	2023-03-23 11:16:25.561357857 +0200
@@ -44,8 +44,8 @@
 
 
 extlinks = {
-    "pypi": ("https://pypi.python.org/pypi/%s", ""),
-    "github": ("https://github.com/%s", "github:/"),
+    "pypi": ("https://pypi.python.org/pypi/%s", "%s"),
+    "github": ("https://github.com/%s", "%s"),
     "issue":  ("https://github.com/behave/behave/issue/%s", "issue #"),
     "youtube": ("https://www.youtube.com/watch?v=%s", "youtube:video="),
     "behave": ("https://github.com/behave/behave", None),
